### DISCUSSION TITLE
Bundling

### DISCUSSION THREAD

**Discussion by rossholdway** [skymethod/denoflare#75] [General]
Hello 👋 

Firstly, thank you for creating Denoflare! I just wanted to ask about bundling. As I understand it, this is how Denoflare runs on Cloudflare Workers, by bundling the code on deploy (and polyfilling the Deno specific API)?

My question is, how is Denoflare doing this? Is it using ‘deno bundle’, and are there plans to move to something else seeing as this has been marked as deprecated for a while now?

https://docs.deno.com/runtime/manual/tools/bundler

Thanks!

**Comment by johnspurlock-skymethod** [comment]
Hi there - Denoflare's been around long enough that it was able to use `Deno.emit()` back when that was included in the Deno runtime.  After that was deprecated, Denoflare switched over to a version that shells out to `deno bundle`, which is what it does today.

At some point before Deno 2.0 ships, Denoflare will switch over to something like esbuild + [esbuild_deno_loader](https://github.com/lucacasonato/esbuild_deno_loader)
